lib/plugins/aws/deploy/compile/events/apiGateway/README.md

```diff
@@ -52,3 +52,40 @@ functions:
                 path: posts/create
                 method: post
 ```
+
+### Http setup with custom authorizer
+You can enable custom authorizers for your HTTP endpoint with the following config:
```

Comment:
Please state more clearly how you set it, e.g. "You can enable custom authorizers for your HTTP endpoint by setting the authorizer in your http event to another function in the same service, as shown in the following example"